src/cmd_opt.ml

```diff
@@ -6,7 +6,7 @@ open Syntax
 
 let optimize_file ~unsafe filename =
   if not @@ Sys.file_exists filename then
-    error_s "file `%s` doesn't exist" filename
+    error_s "Error: file `%s` doesn't exist" filename
```

Comment:
Could you remove the `Error:` ? I used to do this but then ended up in many nested `Error: Error:` when wrapping errors and it quickly becomes a mess. :sweat_smile: